gst/parser/gstttmlparse.c

```diff
@@ -835,7 +835,7 @@ subrip_fix_up_markup (gchar ** p_txt)
         break;
       default:
         GST_ERROR ("unexpected tag '%c' (%s)", *next_tag, next_tag);
-        g_assert_not_reached ();
```

Comment:
`g_assert_not_reached()` is actually an [exception to the rule](https://docs.gtk.org/glib/func.assert_not_reached.html), i.e. they are disabled by `G_DISABLE_ASSERT `, so we can keep those.

I guess that makes this whole PR unnecessary?

Reply:
This is mostly a technicality, keeping `g_assert_not_reached()` likely yields a clearer log but all else is the same.